processor/configuration/src/main/resources/schemas/actions/catalog.json

```diff
@@ -14,5 +14,9 @@
   {
     "id": "webhook_sink_0.1",
     "isConnector": false
+  },
+  {
+    "id": "google_pubsub_sink_0.1",
```

Comment:
Nipick, should the list be sorted alphabetically by id?

Reply:
:+1: done